core/src/test/kotlin/api/RunsRouteIntegrationTest.kt

```diff
@@ -1024,11 +1040,34 @@ class RunsRouteIntegrationTest : AbstractIntegrationTest({
 
                 with(packages.data) {
                     shouldHaveSize(2)
-                    first().identifier.name shouldBe "example"
-                    first().authors shouldHaveSize 2
-                    first().declaredLicenses shouldHaveSize 3
-                    first().processedDeclaredLicense.mappedLicenses shouldHaveSize 2
-                    first().processedDeclaredLicense.unmappedLicenses shouldHaveSize 4
+
+                    with(first()) {
+                        identifier.name shouldBe "example"
+                        authors shouldHaveSize 2
+                        declaredLicenses shouldHaveSize 3
+                        processedDeclaredLicense.mappedLicenses shouldHaveSize 2
+                        processedDeclaredLicense.unmappedLicenses shouldHaveSize 4
+
+                        shortestDependencyPaths shouldNotBeNull {
+                            shouldHaveSize(1)
+
+                            with(first()) {
```

Comment:
`shouldHaveSize(1)` / `first()` calls can be combined into `shouldBeSingleton { ... }`. Similar below.